gems/alchemy_cms/CVE-2018-18307.yml

```diff
@@ -0,0 +1,22 @@
+---
+gem: alchemy_cms
+cve: 2018-18307
+ghsa: 7mj4-2984-955f
+url: http://packetstormsecurity.com/files/149787/Alchemy-CMS-4.1-Stable-Cross-Site-Scripting.html
```

Comment:
Please use either the official advisory URL or the GHSA/NVD URL for the advisory.